src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -176,12 +176,14 @@
           @(Net50AppHostRids);
           osx-arm64;
           linux-s390x;
+          linux-loongarch64;
```

Comment:
Should this be retroactively supported down to .NET 6?  Will that work?  Or is it something that should only work starting with .NET 8 or .NET 9?